app/registration.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * Register components (via a list of glob patterns)
+ *
+ * Copyright © 2015 Magento. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\NonComposerComponentRegistration;
```

Comment:
Actually not as it's not part of autoloading (no class). The namespace is just bulky to not be conflated with anything useful ;)